apps/mobile/src/app/(home)/recover-wallet.tsx

```diff
@@ -190,7 +175,7 @@ export default function RecoverWallet() {
               <ChevronDownIcon color={theme.colors['ink.text-primary']} variant="small" />
             )}
           </TouchableOpacity>
-          {!showAdvancedOptions ? null : (
+          {showAdvancedOptions ? (
```

Comment:
Maybe this can be just `{showAdvancedOptions &&`

Reply:
Good point, prefer that too. Will change.